harness/determined/common/api/authentication.py

```diff
@@ -262,6 +274,10 @@ def _reconfigure_from_store(self, store: dict) -> None:
     def get_active_user(self) -> Optional[str]:
         return self._active_user
 
+    def get_all_user(self) -> List[str]:
+        usernames = [username for (username, _) in self._tokens.items()]
+        return usernames
```

Comment:
a few things:
- make `get_all_users()` plural since it returns multiple.
- there's no need to save your list comprehension as a variable, then return it.  Just return it.
- in python you don't need parens around `for username, _ in x.items()`
- `self._tokens.items()` lets you iterate over `(key, value)`, but you are ignoring the value.  So you can just iterate over `self._tokens` since by default iterating over a dictionary iterates over the keys.

So you could simplify this to be:

```python
def get_all_users(self) -> List[str]:
    return list(self._tokens)
```

Reply:
Thanks, will update it.